draft-ietf-tls-tls13.md

```diff
@@ -3518,15 +3519,15 @@ The following rules apply to the certificates sent by the server:
 
 - The certificate MUST allow the key to be used for signing (i.e., the
   digitalSignature bit MUST be set if the Key Usage extension is present) with
-  a signature scheme indicated in the client's "signature_algorithms" extension.
+  a signature scheme indicated in the client's "signature_algorithms"/"signature_algorithms_cert"
+  extensions (see {{signature-algorithms}}).
```

Comment:
If signature_algorithms_cert is supposed to apply to signatures relevant to the X.509 library, then it only matters for the signature *on* the leaf certificate, not signatures made *by* the leaf cert (right?).  So only signature_algorithms is relevant for this purpose.

Reply:
No, because this applies to chain certs.

Reply:
as well